packages/modelserver-node/src/client/model-server-client.ts

```diff
@@ -431,7 +431,7 @@ class DefaultTransactionContext implements TransactionContext {
     }
 
     // Doc inherited from `Executor` interface
-    async execute(command: ModelServerCommand): Promise<ModelUpdateResult> {
+    async execute(command: ModelServerCommand, modelUri: string): Promise<ModelUpdateResult> {
```

Comment:
Usually `modelUri` or some variant thereof is the first parameter to methods in the client API. Should it be likewise, here? And in many other places where the parameter has been introduced.